sdk/src/auth/index.ts

```diff
@@ -180,7 +180,15 @@ export default class AWF_Auth implements AWF_IAuth {
       try {
         await this._web3.connect(currentProvider);
         await lastValueFrom(this._web3.checkCurrentNetwork());
+        this._globalChannel.next({
+          data: {},
+          event: AUTH_EVENTS.CONNECT_ADDRESS,
+        });
         const address = await lastValueFrom(this._web3.getCurrentAddress());
```

Comment:
replace this with `const address = this. _connectAddress(currentProvider)` and remove event emitted here(to not double them)

Reply:
`        await this._web3.connect(currentProvider);
        await lastValueFrom(this._web3.checkCurrentNetwork());` can be also removed after using `_connectAddress` method